requirements.txt

```diff
@@ -3,11 +3,11 @@ gunicorn<20.0.0
 matplotlib
 numpy
 pandas>=0.24.0
-psutil==5.4.8  # sagemaker-containers requires psutil 5.4.8
+psutil==5.6.6  # sagemaker-containers requires psutil 5.4.8
```

Comment:
Is the comment `sagemaker-containers requires psutil 5.4.8` still true?

Reply:
actually, sagemaker-containers is using psutil `5.6.7` in their` 2.8.3` version. updated PR.